paddle/scripts/docker/build.sh

```diff
@@ -83,11 +108,25 @@ if [[ ${WOBOQ:-OFF} == 'ON' ]]; then
         -o $WOBOQ_OUT \
         -p paddle:/paddle
     /woboq/indexgenerator/codebrowser_indexgenerator $WOBOQ_OUT
-    cd /woboq
-    make clean
 fi
 
-paddle version
+# generate deb package for current build
+# FIXME(typhoonzero): should we remove paddle/scripts/deb ?
+# FIXME: CPACK_DEBIAN_PACKAGE_DEPENDS removes all dev dependencies, must
+# install them in docker
+cat <<EOF
+========================================
+Generating .deb package ...
+========================================
+EOF
+cpack -D CPACK_GENERATOR='DEB' -D CPACK_DEBIAN_PACKAGE_DEPENDS="" ..
```

Comment:
Should remove `-D CPACK_DEBIAN_PACKAGE_DEPENDS=""`.  I added this before because the base image is `python:2.7.13-slim` which installs python by building from source code. Install Paddle `wheel` package will install another version of python in the image, so I removed the dependencies. Now we are using `apt-get` to install python and python header files, so dependencies will not break when install.